pkg/instance/tshark.go

```diff
@@ -0,0 +1,49 @@
+package instance
+
+import (
+	"context"
+)
+
+func (i *Instance) createTsharkCollectorInstance(ctx context.Context) (*Instance, error) {
+	tsharkCollector, err := New("tshark-collector", i.SystemDependencies)
+	if err != nil {
+		return nil, err
+	}
+	if tsharkCollector.SetImage(ctx, "ttl.sh/68d00fc2-c488-4e3a-a787-5c0e8c8135d9:24h") != nil {
```

Comment:
I think there is a typo here. this image lasts for 24 hours and then will be deleted.